robots/preservation/transfer_object.rb

```diff
@@ -0,0 +1,89 @@
+
+module Preservation
+  # Robot for transferring objects from the DOR export area to the SDR deposit area
+  class TransferObject < Base
+    def initialize(opts = {})
+      super(REPOSITORY, WORKFLOW_NAME, 'transfer-object', opts)
+    end
+
+    attr_reader :druid
+
+    def perform(druid)
+      @druid = druid
+      transfer_object
+    end
+
+    private
+
+    # Transfer and untar the object from the DOR export area to the SDR deposit area.
+    #   Note: POSIX tar has a limit of 100 chars in a filename
+    #     some implementations of gnu TAR work around this by adding a ././@LongLink file containing the full name
+    #     See: http://www.delorie.com/gnu/docs/tar/tar_114.html
+    #      http://stackoverflow.com/questions/2078778/what-exactly-is-the-gnu-tar-longlink-trick
+    #      http://www.gnu.org/software/tar/manual/html_section/Portability.html
+    #   Also, beware of incompatibilities between BSD tar and other TAR formats
+    #     regarding the handling of vendor extended attributes.
+    #     See: http://xorl.wordpress.com/2012/05/15/admin-mistakes-gnu-bsd-tar-and-posix-compatibility/
+    def transfer_object
+      LyberCore::Log.debug("transfer_object #{druid} starting")
+      prereqs_for_transfer
+      deposit_dir = prepare_deposit_dir
+      Base.execute_shell_command(tarpipe_command(deposit_dir))
```

Comment:
slightly nitpicky, but what about `self.class.` instead of `Base.`?  slightly longer, but allows for inheritance to work automatically.